src/modules/involvementAPI/addLikes.js

```diff
@@ -0,0 +1,16 @@
+import { getLikes } from './getLikes.js';
+
+const addLikes = async () => {
+  const likes = await getLikes();
+  const likesContainer = document.querySelectorAll('.likes');
+  likesContainer.forEach((like) => {
+    const id = like.parentElement.parentElement.parentElement.parentElement
```

Comment:
- [ ] And yes, what was mentioned by the previous reviewer is true, you need to add tests for all counter functions as per the requirement, I would suggest you implement the change as he mentioned before(test cases) and resubmit your work.
<img width="889" alt="1" src="https://user-images.githubusercontent.com/77843050/219642845-5813d498-848c-4d42-850c-77a4f4389a02.png">